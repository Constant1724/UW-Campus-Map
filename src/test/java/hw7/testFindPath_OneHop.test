# START_TEST
# NAME OneHop
# test a graph can be loaded and correctly find the path from A to B.

# Create a graph and load data
LoadGraph heros staffSuperheroes.tsv


FindPath heros Ernst-the-Bicycling-Wizard Perkins-the-Magical-Singing-Instructor

FindPath heros Ernst-the-Bicycling-Wizard Notkin-of-the-Superhuman-Beard

# END_TEST