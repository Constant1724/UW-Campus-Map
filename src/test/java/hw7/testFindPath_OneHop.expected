# START_TEST
# NAME OneHop
# test a graph can be loaded and correctly find the path from A to B.

# Create a graph and load data
loaded graph heros


path from Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor:
Ernst-the-Bicycling-Wizard to Perkins-the-Magical-Singing-Instructor with weight 1.000
total cost: 1.000

path from Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard:
Ernst-the-Bicycling-Wizard to Notkin-of-the-Superhuman-Beard with weight 0.500
total cost: 0.500

# END_TEST